lib/blocks.php

```diff
@@ -397,299 +357,3 @@ function gutenberg_register_legacy_social_link_blocks() {
 }
 
 add_action( 'init', 'gutenberg_register_legacy_social_link_blocks' );
-
-/**
- * Checks whether the current block type supports the feature requested.
- *
- * @param WP_Block_Type $block_type Block type to check for support.
- * @param array         $feature    Path of the feature to check support for.
- * @param mixed         $default    Fallback value for feature support, defaults to false.
- *
- * @return boolean                  Whether or not the feature is supported.
- */
-function gutenberg_block_has_support( $block_type, $feature, $default = false ) {
```

Comment:
https://developer.wordpress.org/reference/functions/block_has_support/ was introduced in WordPress 5.8 and that is the minimum supported version for the plugin.